### ISSUE TITLE
Server IP null during loading of plugins

### ISSUE THREAD

**Issue by AuroraLS3** [GlowstoneMC/Glowstone#431]
Title: Server IP is null while loading plugins

Glowstone build: Glowstone version 2017.2.0.20-SNAPSHOT-"8a57f8a"-MC1.11.2

I decided to test if my Bukkit plugins work on Glowstone, and it appears that IP is null during boot when plugins are loading

```
onEnable() {  
String ip = getServer().getIP();  
ip.isEmpty(); <-- Throws NPE  
}
```

---
Error thrown on console
```
16:15:18 [SEVERE] Error occurred while enabling Plan v2.7.2 (Is it up to date?)
java.lang.NullPointerException
        at main.java.com.djrapitops.plan.Plan.onEnable(Plan.java:145)
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:271)
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316)
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:415)
        at net.glowstone.GlowServer.enablePlugins(GlowServer.java:796)
        at net.glowstone.GlowServer.start(GlowServer.java:512)
        at net.glowstone.GlowServer.run(GlowServer.java:432)
        at net.glowstone.GlowServer.main(GlowServer.java:305)
```
Lines 143 to 145 in Plan.java (My plugin)
```
        Server server = getServer();
        String ip = server.getIp();
        if (ip.isEmpty()) {
```

**Comment by gdude2002** [comment]
I'm guessing Craftbukkit just returns the empty string here if no IP is set in the config?

**Comment by AuroraLS3** [comment]
Most likely, I've got server-ip=localhost in server.properties though

**Comment by gdude2002** [comment]
Glowstone doesn't use `server.properties`; did you mean the `.yml` file in `config/`?

**Comment by AuroraLS3** [comment]
Ah I just copied my other spigot folder and put glowstone on it, but it appears that the glowstone.yml reads:
```
server:
    ip: localhost
    port: 25565
```

**Comment by gdude2002** [comment]
Alright, well this is probably a quick fix, but I'm unable to do any testing right now as I'm at work, so I'm going to delegate it to @momothereal

**Comment by mastercoms** [comment]
Yes, Glowstone migrates settings from `bukkit.yml` and `server.properties`.

I'll make a test plugin to see CraftBukkit behavior on this.

**Comment by aramperes** [comment]
This is potentially fixed in 5739b7b688de3875089331aa0e65750fa147b62c, please mind testing it and tell us the results.

**Comment by AuroraLS3** [comment]
Version 9e0e517 did not throw NPE when loading the plugin.
